components/lib/autocomplete/AutoComplete.js

```diff
@@ -446,7 +446,7 @@ export const AutoComplete = React.memo(
         };
 
         const getScrollableElement = () => {
-            return virtualScrollerRef.current ? overlayRef.current.firstChild : overlayRef.current;
+            return virtualScrollerRef.current ? overlayRef.current.firstChild : overlayRef.current.firstChild;
```

Comment:
This makes both sides of the ternary the same and will break other scenarios.  How about...
```suggestion
            return virtualScrollerRef.current || props.optionGroupLabel ? overlayRef.current.firstChild : overlayRef.current;
```

Reply:
Hi @melloware, you are right the both side is same for the ternary operator. But the issue is there with all kind of autocompletes not only grouped. Please check my latest changes.

Reply:
let me test in the showcase.

Reply:
Yep your fix seems to be working for all the showcase examples